app/scripts/views/mixins/open-confirmation-email-mixin.js

```diff
@@ -0,0 +1,96 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+/**
+ * A view mixin that helps confirm* views show the "Open Gmail" button
+ * directly from FxA.
+ */
+define(function (require, exports, module) {
+  'use strict';
+
+  var BaseView = require('views/base');
+  var t = BaseView.t;
+
+  return {
+    events: {
+      'click #open-webmail': '_webmailTabOpened'
+    },
+
+    /**
+     * Get the webmail URL for the given email
+     *
+     * @param {string} email
+     * @returns {string}
+     */
+
+    _isGmail: function (email) {
+      return /@gmail\.com$/.test(email);
+    },
+
+    _isHotmail: function (email) {
+      return /@hotmail\.com$/.test(email);
+    },
+
+    _isYahoo: function (email) {
+      return /@yahoo\.com$/.test(email);
+    },
+
+    getWebmailLink: function (email) {
+      var linkToBeReturned;
```

Comment:
nit: `ToBeReturned` in `getWebmailLink` and `getWebmailButtonText` is a bit strange. maybe just `webmailLink` and `webmailText`. At least one reason against `toBeReturned` -> what if you refactor in the future where the variable is modified again and you have to rename `toBeReturned` because it's not the var that is returned anymore.